tests/test_backends.py

```diff
@@ -239,15 +239,13 @@ def test_build_vllm_cmd_with_defaults(tmp_path: pathlib.Path):
     chat_template.touch()
     vllm_args: list[str] = []
     expected_cmd = [
-        sys.executable,
-        "-m",
-        "vllm.entrypoints.openai.api_server",
+        "vllm",
+        "serve",
+        str(model_path),
```

Comment:
@russellb why was `str(model_path)` used here and `os.fspath(model_path)` used in the test below?